### ISSUE TITLE
Analysis of Loss and Accuracy over epochs(more general question)

### ISSUE THREAD

**Issue by paipai880429** [fchollet/keras#481]
I would like to ask a more general question about the analysis of historical loss and accuracy over epochs. I have used keras for training a ConvNet regression problem. The loss and accuracy visualization is below. The loss seems to be OK. But look at the accuracy figure, the valid does not seem to be reasonable. What problem may I occur? Overfitting? Underfitting ?

![image](https://cloud.githubusercontent.com/assets/3724085/9079675/6b199d82-3b7f-11e5-97b3-047d7aae556b.png)
![image](https://cloud.githubusercontent.com/assets/3724085/9079683/775e59e8-3b7f-11e5-9bc4-7ca78aa37f94.png)

**Comment by paipai880429** [comment]
@fchollet Would like to hear your voice : )

**Comment by fchollet** [comment]
It looks pretty standard. The reason for the ceiling at 0.7 is to be found in your data / task, not in the Keras training procedure.

In general, you will always see training losses higher than the validation losses (at first) if you are using dropout. It means you are still underfitting. When validation loss becomes higher than training loss, you are overfitting.
